src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -25,14 +25,16 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import io.github.cdimascio.dotenv.Dotenv;
 
 /** BusinessesService object representing all businesses 
 * components of the webapp.
  **/
 public class BusinessesService {
-  
+  private Dotenv dotenv = Dotenv.configure().load();
+
   private List<Listing> allBusinesses;
-  private final String KEY = "REDACTED";
+  private final String KEY = dotenv.get("PlacesAPI");
```

Comment:
Super nit: Since the key can be used for any API throughout the webapp, a more general name might be helpful in the future

Reply:
Isn't this key specifically only for the places or maps feature. Like I am assuming a different api key is for the Custom Search API for example

Reply:
We can use the same API key across the whole application for any Google API. I think the best practice is not share keys across different apps, so we should be okay.

Reply:
Yes you are right, the same Key is for all the different API's Google offers. I just changed the name of it, to APIKey if that is better